src/metabase/util/ssh.clj

```diff
@@ -22,6 +23,12 @@
    (org.apache.sshd.common.util.security SecurityUtils)
    (org.apache.sshd.server.forward AcceptAllForwardingFilter)))
 
+(defsetting ssh-heartbeat-interval-sec
+  (deferred-tru "Controls how often the heartbeats are sent when an SSH tunnel is established (in seconds).")
+  :visibility :public
+  :type       :integer
+  :default    180)
```

Comment:
I had to move from public_settings to resolve a cyclic dependency. I think it's a better place anyway, since it's only used in line 59 below.